db/migrate/20200415095035_migrate_image_url_to_image_upload_id.rb

```diff
@@ -0,0 +1,95 @@
+# frozen_string_literal: true
+
+class MigrateImageUrlToImageUploadId < ActiveRecord::Migration[6.0]
```

Comment:
feels like a post migration is what we want here, no need to hold off the deploy for this, just run it after all app servers have the new code.

Reply:
If we run this as a post-migration, then sites will stop serving opengraph images on all topics for a few minutes. I guess that's ok... but if we keep it as a pre-deploy migration then the change should be seamless. I think it should only take a few seconds.